buildSrc/src/main/kotlin/ort-server-base-conventions.gradle.kts

```diff
@@ -20,6 +20,14 @@
 repositories {
     mavenCentral()
 
+    maven {
+        name = "localOrt"
+        url = uri(mavenLocal().url)
+        content {
+            includeGroupByRegex("org\\.ossreviewtoolkit(\\..*)?")
+        }
+    }
+
```

Comment:
How is this different from

    mavenLocal() {
        name = "localOrt"

        content {
            includeGroupByRegex("org\\.ossreviewtoolkit(\\..*)?")
        }
    }